### PR TITLE
Add templates for Python

### PR DESCRIPTION
**Pull request by elprans**

### CODE REVIEW

**Comment by elprans** [comment]
@asavaritayal @soninaren, the portal doesn't seem to have support for Python templates yet, so I couldn't test this there:

screenshot_20180313_134644

**Comment by asavaritayal** [comment]
@elprans @soninaren Is this ready to merge?

File: Functions.Templates/Templates/BlobTrigger-Python/__init__.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import logging
+
+import azure.functions as azf
```
Comment (asavaritayal) [inline Functions.Templates/Templates/BlobTrigger-Python/__init__.py]: Can we rename `azf` to `func` instead? Most functions users are already familiar with the alias and it's easy to read.

File: Functions.Templates/Templates/HttpTrigger-Python/function.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        {
+            "authLevel": "function",
+            "type": "httpTrigger",
+            "direction": "in",
+            "name": "req"
+        },
+        {
+            "type": "http",
+            "direction": "out",
+            "name": "res"
```
Comment (asavaritayal) [inline Functions.Templates/Templates/HttpTrigger-Python/function.json]: This should be `"name": "$return"`. The current function fails.

File: Functions.Templates/Templates/BlobTrigger-Python/function.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+    "scriptFile": "__init__.py",
+    "disabled": false,
+    "bindings": [
+        {
+            "name": "my_blob",
```
Comment (asavaritayal) [inline Functions.Templates/Templates/BlobTrigger-Python/function.json]: The functions host yells if you give it an `_` in the `name`. Can you change this to `myblob`, along with any other instances in the code?

**Comment by elprans** [comment]
@asavaritayal Thanks for the review.  I've pushed the fixes.

**Comment by elprans** [comment]
Added two more templates: `QueueTrigger` and `TimerTrigger`.

File: Functions.Templates/Templates/BlobTrigger-Python/readme.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# BlobTrigger - JavaScript
```
Comment (soninaren) [inline Functions.Templates/Templates/BlobTrigger-Python/readme.md]: +# BlobTrigger - Python

File: Functions.Templates/Templates/BlobTrigger-Python/readme.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
+# BlobTrigger - JavaScript
+
+The `BlobTrigger` makes it incredibly easy to react to new Blobs inside of Azure Blob Storage. This sample demonstrates a simple use case of processing data from a given Blob using JavaScript.
```
Comment (soninaren) [inline Functions.Templates/Templates/BlobTrigger-Python/readme.md]: replace javascript with python
Blob using Python.

Comment (elprans) [reply Functions.Templates/Templates/BlobTrigger-Python/readme.md]: Fixed

File: Functions.Templates/Templates/BlobTrigger-Python/readme.md (hunk 4 above)
Comment (elprans) [reply Functions.Templates/Templates/BlobTrigger-Python/readme.md]: Fixed

**Comment by asavaritayal** [comment]
@elprans , @soninaren can you look into the conflicts?

**Comment by elprans** [comment]
@asavaritayal  Rebased.

File: Functions.Templates/Templates/HttpTrigger-Python/__init__.py [hunk 6]
```diff
@@ -0,0 +1,24 @@
+import logging
+
+import azure.functions as func
+
+
```
[the remark this answers is not in the document]
Comment (elprans) [reply Functions.Templates/Templates/HttpTrigger-Python/__init__.py]: Two empty lines are per standard (flake8) Python lint rules.

File: Functions.Templates/Templates/BlobTrigger-Python/__init__.py [hunk 7]
```diff
@@ -0,0 +1,9 @@
+import logging
+
+import azure.functions as func
+
+
```
[the remark this answers is not in the document]
Comment (elprans) [reply Functions.Templates/Templates/BlobTrigger-Python/__init__.py]: Two empty lines here are a lint requirement for python.

File: Functions.Templates/Templates/HttpTrigger-Python/__init__.py (hunk 6 above)
[the remark this answers is not in the document]
Comment (elprans) [reply Functions.Templates/Templates/HttpTrigger-Python/__init__.py]: Two empty lines here are a lint requirement for python.

File: Functions.Templates/Templates/QueueTrigger-Python/__init__.py [hunk 8]
```diff
@@ -0,0 +1,8 @@
+import logging
+
+import azure.functions as func
+
+
```
[the remark this answers is not in the document]
Comment (elprans) [reply Functions.Templates/Templates/QueueTrigger-Python/__init__.py]: Two empty lines here are a lint requirement for python.

File: Functions.Templates/Templates/TimerTrigger-Python/__init__.py [hunk 9]
```diff
@@ -0,0 +1,14 @@
+import datetime
+import logging
+
+import azure.functions as func
+
+
```
[the remark this answers is not in the document]
Comment (elprans) [reply Functions.Templates/Templates/TimerTrigger-Python/__init__.py]: Two empty lines here are a lint requirement for python.

**Comment by soninaren** [comment]
Merging manually

**Comment by soninaren** [comment]
Merged this PR here
https://github.com/Azure/azure-functions-templates/commit/d78c38980ab1d11d1e2d90a1bf867c630834e81c

**Pull request closed without merging**